devtools/server/actors/replay/auth.js

```diff
@@ -65,4 +114,142 @@ function tokenExpiration(token) {
   }
   const exp = userInfo.payload?.exp;
   return typeof exp === "number" ? exp * 1000 : null;
-}
\ No newline at end of file
+}
+
+let authChannel;
+
+function handleAuthChannelMessage(_id, message, target) {
+  const { type } = message;
+  if (type === "connect") {
+    deferredAccessToken.promise.then(token => {
+      if (authChannel) {
+        authChannel.send({ token }, target);
+      }
+    })
+  // TODO [ryanjduffy]: Add support for app login to use the browser auth flow
+  // } else if (type === "login") {
+  //   openSigninPage();
```

Comment:
This commented out code is a little strange, can you remove it and maybe expand the comment above it?